test/factories/partnerships.rb

```diff
@@ -0,0 +1,12 @@
+FactoryBot.define do
+  factory :partnership do
+    association :provider
+
+    accredited_provider { create(:provider, :accredited) }
+    duration { (Date.new(Time.zone.now.year, 1, 1)..) }
```

Comment:
can we make a trait out of this...

looks like ongoing vs closed end partnership